fms2_io/include/get_global_attribute.inc

```diff
@@ -63,7 +63,8 @@ subroutine get_global_attribute_0d(fileobj, &
                                    trim(attribute_name), &
                                    attribute_value)
             class default
-                call error("get_global_attribute_0d: unsupported type for "//trim(attribute_name)//" for file: "//trim(fileobj%path)//"")
+                call error("get_global_attribute_0d: unsupported type for "//&
+                trim(attribute_name)//" for file: "//trim(fileobj%path)//"")
```

Comment:
Since this is within a string, probably should probably use an & right in front of the continued line as well.  Same for the rest of the instances in the file at lines 104, 146, & 178.